account_statement_import_txt_xlsx/readme/CONTRIBUTORS.rst

```diff
@@ -1,5 +1,6 @@
 * Alexis de Lattre <alexis.delattre@akretion.com>
 * Sebastien BEAU <sebastien.beau@akretion.com>
+* Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
```

Comment:
The improvement commit seems to be from zaoral, but I don't see her here (nor you).